### PR TITLE
update repo docs without PR

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
- push directly to gh-pages branch instead of making PRs
- fixed labeler permissions for actions in forks
- syncs docs with ones in VM repo. fixes https://github.com/VictoriaMetrics/helm-charts/issues/526

### CODE REVIEW

File: charts/victoria-metrics-agent/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+weight: 2
+title: Victoria Metrics Agent
```
Comment (Haleygo) [inline charts/victoria-metrics-agent/_index.md:3]: "Victoria Metrics" should be "VictoriaMetrics", would you mind fix them all in charts?

**Review by Haleygo — APPROVED** [review]

File: charts/victoria-metrics-agent/_index.md (hunk 1 above)
Comment (AndrewChubatiuk) [reply charts/victoria-metrics-agent/_index.md]: done

File: charts/victoria-metrics-operator/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+weight: 9
```
Comment (Haleygo) [inline charts/victoria-metrics-operator/_index.md:2]: It has the same weight as the previous K8s Stack.

Comment (AndrewChubatiuk) [reply charts/victoria-metrics-operator/_index.md]: updated it as well

**Review by Haleygo — APPROVED** [review]
LGTM!

**Pull request merged**